src/main/java/Duke.java

```diff
@@ -1,10 +1,158 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static Task[] list = new Task[100];
+    public static int taskLength = 0;
+    public static final String NEW_LINE = "____________________________________________________________\n";
+
+    //Create a new task
+    public static void createTask(String task) {
+        list[taskLength] = new Deadline(task);
+        taskLength++;
+        System.out.println(NEW_LINE + "Okay, I've added: " + task + "\n" + NEW_LINE);
+    }
+
+    //Handle different types of tasks
+    public static void handleTasks(String input) throws EmptyTaskException{
+        int index = input.indexOf(" ");
+        String taskType = input.substring(0, index);
+        String taskContent = input.substring(index);
+        String taskFinal;
+
+        switch (taskType) {
+        case "deadline":
+            try {
+                //Empty deadline exception
+                String deadlineContent = taskContent.split("/by ")[0];
+                if (deadlineContent.trim().isEmpty()) {
+                    throw new EmptyTaskException();
+                }
+                else {
+                    String deadlineTiming = taskContent.split("/by ")[1];
+                    taskFinal = taskContent.split("/")[0] + " (by:" + deadlineTiming + ")";
+                    createTask(taskFinal);
+                }
+            }
+            catch (IndexOutOfBoundsException e) {
+                System.out.println("Cannot parse deadline start date!\n"
+                + "Format: deadline [task] /by [time]");
+            }
+            break;
+
+        case "event":
+            try {
+                //empty event exception
+                String eventContent = taskContent.split("/from ")[0];
+                if (eventContent.trim().isEmpty()) {
+                    throw new EmptyTaskException();
+                }
+                else {
+                    String eventTiming = taskContent.split("/from ")[1];
+                    String eventFrom = eventTiming.split("/to ")[0];
+                    String eventTo = eventTiming.split("/to ")[1];
+                    taskFinal = taskContent.split("/")[0] + " (from: " + eventFrom + "to: " + eventTo + ")";
+                    createTask(taskFinal);
+                }
+            }
+            catch (IndexOutOfBoundsException e) {
+                System.out.println("Cannot parse event start/end date!\n"
+                +  "Format: event [task] /from [start time] /to [end time]");
```

Comment:
Try not to use magic literals e.g. magic numbers and magic string. Consider giving it a meaning by creating a constant and use the constant in your code.